docs/CONFIGURATION.md

```diff
@@ -1,46 +1,46 @@
-# Ubersystem Configuration
```

Comment:
Is GitHub just being weird about file movements? Normally it doesn't generate full diffs for file moves.